docs/examples/use_cases/tensorflow/efficientdet/utils.py

```diff
@@ -30,6 +28,94 @@
 # pylint: disable=logging-format-interpolation
 
 
+class PipelineType(Enum):
+    syntetic = 0
+    tensorflow = 1
+    dali_cpu = 2
+    dali_gpu = 3
+
+
+class InputType(Enum):
+    tfrecord = 0
+    coco = 1
+
+
+def dict_to_namedtuple(字典):
+    NamedTuple = namedtuple("NamedTuple", 字典.keys())
+    return NamedTuple._make(字典.values())
```

Comment:
Google Translate to the rescue!
```suggestion
def dict_to_namedtuple(dictionary):
    NamedTuple = namedtuple("NamedTuple", dictionary.keys())
    return NamedTuple._make(dictionary.values())
```

Reply:
I agree here. We shouldn't get outside of ascii.

Reply:
Fixed ;)